client/my-sites/domains/domain-management/dns/dns-records.jsx

```diff
@@ -207,6 +214,29 @@ class DnsRecords extends Component {
 			return null;
 		}
 
+		let translatedMessage = translate(
+			'Your domain is not using default A records. This means it may not be pointing to your WordPress.com site correctly. To restore default A records, click on the three dots menu and select "Restore default A records". {{defaultRecordsLink}}Learn more{{/defaultRecordsLink}}.',
+			{
+				components: {
+					defaultRecordsLink: (
+						<InlineSupportLink supportContext="dns_default_records" showIcon={ false } />
+					),
+				},
+			}
+		);
+		if ( selectedDomain?.isGravatarDomain ) {
+			translatedMessage = translate(
+				'Your domain is not using default A records. This means it may not be pointing to your Gravatar Profile correctly. To restore default A records, click on the three dots menu and select "Restore default A records". {{defaultRecordsLink}}Learn more{{/defaultRecordsLink}}.',
```

Comment:
Would that make sense?

```
const affectedComponent = ( selectedDomain?.isGravatarDomain ) ? 'Gravatar Profile' : 'WordPress.com site';
let translatedMessage = translate(
			'Your domain is not using default A records. This means it may not be pointing to your %(affectedComponent)s correctly. To restore default A records, click on the three dots menu and select "Restore default A records". {{defaultRecordsLink}}Learn more{{/defaultRecordsLink}}.',
			{
				args: { affectedComponent },
				components: {
					defaultRecordsLink: (
						<InlineSupportLink supportContext="dns_default_records" showIcon={ false } />
					),
				},
			}
		);
```